### PR TITLE
Fix Thread for "optional" features and little optimalization

### PR DESCRIPTION
**Pull request by TheFantomas**
Thanks Tinky124 for inspiration in DisableVehicleRewards

### CODE REVIEW

**Review by CsokiHUN — APPROVED** [review]
Looks good, but i think we use [esx:enteredVehicle](https://github.com/esx-framework/esx_core/blob/6ee4be79ddf16cd36288c3e7bcdd5d32d5a848d2/%5Bcore%5D/es_extended/client/modules/actions.lua#L69) event

**Comment by TheFantomas** [comment]
> Looks good, but i think we use [esx:enteredVehicle](https://github.com/esx-framework/esx_core/blob/6ee4be79ddf16cd36288c3e7bcdd5d32d5a848d2/%5Bcore%5D/es_extended/client/modules/actions.lua#L69) event

Yep... I will fix it :D

File: [core]/es_extended/client/main.lua [hunk 1]
```diff
@@ -57,66 +57,61 @@ AddEventHandler('esx:playerLoaded', function(xPlayer, isNew, skin)
...
+	end
 
-			if Config.DisableWeaponWheel then
-				Sleep = false
-				BlockWeaponWheelThisFrame()
-				DisableControlAction(0, 37,true)
-			end
+	-- DisableVehicleRewards
+	if Config.DisableVehicleRewards then
+		AddEventHandler('esx:enteredVehicle', function(vehicle, plate, seat, displayName, netId)
```
Comment (Gellipapa) [inline [core]/es_extended/client/main.lua:79]: I think it's enough to look at the emergency vehicle class or other vehicle class can you give a reward?

**Review by Gellipapa — APPROVED** [review]

**Comment by Tinky124** [comment]
> Thanks Tinky124 for inspiration in DisableVehicleRewards

<3

**Review by Tinky124 — APPROVED** [review]

**Review by Rav3n95 — APPROVED** [review]

**Pull request merged**